code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs

```diff
@@ -39,7 +39,7 @@ import Drasil.GamePhysics.Concepts (gamePhysics, acronyms, threeD, twoD)
 import Drasil.GamePhysics.DataDefs (dataDefs)
 import Drasil.GamePhysics.Goals (goals)
 import Drasil.GamePhysics.IMods (iMods, instModIntro)
-import Drasil.GamePhysics.References (citations, koothoor2013, smithLai2005)
+import Drasil.GamePhysics.References (citations, koothoor2013, smithEtAl2007, smithLai2005, smithKoothoor2016)
```

Comment:
Try to keep line lengths below 80 characters where possible:
```suggestion
import Drasil.GamePhysics.References (citations, koothoor2013, smithEtAl2007,
  smithLai2005, smithKoothoor2016)
```

Reply:
@harmanpreet-sagar, to help with line length, you can install Rewrap in VSCode (https://stackoverflow.com/questions/43122175/automatically-hard-wrap-lines-at-column-in-vscode).  I believe this is one of the steps in setting up a new workspace, but you might not have set the line length.  The usual suggestion is 80 characters.  Use Alt+Q to rewrap a paragraph with hard returns.